src/app/loginComponents/accounts/external/accounts.component.scss

```diff
@@ -0,0 +1,45 @@
+@use '~@angular/material' as mat;
+@import '/src/materialColorScheme.scss';
+
+.account-logo {
+  width: 3.5em;
+  margin-right: 1em;
+}
+
+.token-form-field {
+  width: 300px;
+  font-size: 12px;
+}
+
+.color {
+  color: mat.get-color-from-palette($dockstore-app-accent-1, darker);
+}
+
+.mat-badge-medium.mat-badge-above .mat-badge-content {
+  top: 0px;
+  left: 40px;
+  background: #26bfa5;
+}
+
+.card-style {
+  height: 100%;
+}
+
+.button-size {
+  transform-origin: 100% 50%;
+  transform: scale(0.85);
+}
+
+.unlink-button {
+  background: mat.get-color-from-palette($dockstore-app-accent-1, darker);
```

Comment:
seeing this color show up in many places leads me to believe that the styles.scss .accent-1-dark's comment is incorrect. Change that comment.